packages/vue/src/useUploadThing.ts

```diff
@@ -129,7 +126,13 @@ export const INTERNAL_uploadthingHookGen = <
     return {
       startUpload,
       isUploading,
-      permittedFileInfo,
+      routeConfig,
+      /**
+       * @deprecated Use `routeConfig` instead
+       */
+      permittedFileInfo: routeConfig
+        ? { slug: endpoint, config: routeConfig }
+        : undefined,
```

Comment:
think this must be a `computed` to be updated

Reply:
I think you are correct , will fix